httpserver/service_node.cpp

```diff
@@ -363,27 +395,31 @@ void ServiceNode::purge_outdated() {
     return;
 }
 
-void ServiceNode::process_push_all(std::shared_ptr<std::string> blob) {
-
+void ServiceNode::process_push_batch(const std::string& blob) {
     // Note: we only receive batches on bootstrap (new swarm/new snode)
 
-    /// This should already be checked, but just to be sure
-    if (!blob || *blob == "")
+    if (blob == "")
```

Comment:
`blob.empty()` might be more expressive about the intention?

Reply:
changed